client/router.js

```diff
@@ -230,8 +230,8 @@ ClientRouter.prototype.exitApp = function (path) {
 }
 
 ClientRouter.prototype.noRelativePath = function (path) {
-  //if it's not a full url or already has a leading slash
-  if (path.indexOf('://') == -1 && path.charAt(0) !== '/') {
+  //if path doesn't have a protocol and lacks a leading slash
+  if (/^[a-z]+:/.test(path) === false && path.charAt(0) !== '/') {
```

Comment:
Hmm, this won't match if someone does something like `HTTP://` - which should technically work. (even though it's a bit weird) shall we add `/i` to it?

I think the `:` regex should be a valid test since anywhere else it *should* be urlencoded though.